### DISCUSSION TITLE
Upgrade to latest in Umbraco 8.14

### DISCUSSION THREAD

**Discussion by jonroberts24** [nathanwoulfe/Plumber-2#88] [General]
Hi,
Is Plumber tested and working in the latest Plumber and on the latest Umbraco 8.14 version?
Thanks

**Comment by nathanwoulfe** [comment]
Hey @jonroberts24, not formally - I haven't tried 8.14 yet. Have though spent the last few days running it against 8.13.1 and a large database (200+ user groups, 1000s of permissions) and all looks fine. 

That said, what was going to be a quick 1.6.3 release is turning into 1.7.0, with some improvements to group and permission management. 

Long story short, it should be fine in 8.14,  but might be worth waiting a week or so (if possible) for the next minor. Beta is up now, just needs a little polish.
